src/Detector/Hugo/HugoDetector.cs

```diff
@@ -66,31 +66,43 @@ private bool IsHugoApp(ISourceRepo sourceRepo, out string appDirectory)
             appDirectory = string.Empty;
 
             // Search for config.toml
-            if (sourceRepo.FileExists(HugoConstants.TomlFileName)
-                && this.IsHugoTomlFile(sourceRepo, HugoConstants.TomlFileName))
+            foreach (string tomlFileName in HugoConstants.TomlFileName)
             {
-                return true;
+                if (sourceRepo.FileExists(tomlFileName)
+                && this.IsHugoTomlFile(sourceRepo, tomlFileName))
+                {
+                    return true;
+                }
             }
 
             // Search for config.yml
-            if (sourceRepo.FileExists(HugoConstants.YmlFileName)
-                && this.IsHugoYamlFile(sourceRepo, HugoConstants.YmlFileName))
+            foreach (string ymlFileName in HugoConstants.YmlFileName)
             {
-                return true;
+                if (sourceRepo.FileExists(ymlFileName)
+                && this.IsHugoYamlFile(sourceRepo, ymlFileName))
+                {
+                    return true;
+                }
             }
 
             // Search for config.yaml
-            if (sourceRepo.FileExists(HugoConstants.YamlFileName)
-                && this.IsHugoYamlFile(sourceRepo, HugoConstants.YamlFileName))
+            foreach (string yamlFileName in HugoConstants.YamlFileName)
             {
-                return true;
+                if (sourceRepo.FileExists(yamlFileName)
+                && this.IsHugoYamlFile(sourceRepo, yamlFileName))
+                {
+                    return true;
+                }
             }
 
             // Search for config.json
-            if (sourceRepo.FileExists(HugoConstants.JsonFileName)
-                && this.IsHugoYamlFile(sourceRepo, HugoConstants.JsonFileName))
+            foreach (string jsonFileName in HugoConstants.JsonFileName)
             {
-                return true;
+                if (sourceRepo.FileExists(jsonFileName)
+                && this.IsHugoYamlFile(sourceRepo, jsonFileName))
```

Comment:
This looks like an issue from before, but let's change this call to `this.IsHugeJsonFile`